pkg/controller/pod_volume_backup_controller_test.go

```diff
@@ -1,173 +0,0 @@
-/*
```

Comment:
Why was this test file removed?

Reply:
Sorry for the confusion by this PR, this PR is for reproduce an issue, I will close it.